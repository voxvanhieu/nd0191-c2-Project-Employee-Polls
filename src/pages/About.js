import {
    Tabs,
    Tab,
    Listbox,
    ListboxItem,
    Dropdown,
    DropdownTrigger,
    DropdownMenu,
    DropdownItem,
    Button,
    Link,
} from "@nextui-org/react";
import { Fragment } from "react";

export { About }

function About() {
    return (
        <div className="container mx-auto">
            <div className="flex flex-col items-center justify-center min-h-screen text-center">
                <h1 className="text-5xl font-bold mb-5 text-primary-600">Udacity Lab Employee Polls</h1>
                <p className="text-lg text-justify">
                    The final lab exercise of Udacity's React Nanodegree is a comprehensive project that encapsulates the core principles of React and Redux in building a dynamic and interactive voting portal. In this application, users can log in with their personal accounts to engage in a community-driven polling environment. They have the ability to post new polls, reflecting current topics or queries they are curious about, and vote on existing ones, contributing to the collective decision-making process. The user interface is designed to be intuitive, allowing users to easily navigate between polls they have participated in and those they have yet to explore. The polls are sorted by creation date, ensuring a seamless and up-to-date experience. This project not only tests the students' ability to implement state management and component-based architecture but also their skills in creating a user-friendly platform that encourages active participation and interaction within the company's ecosystem.
                </p>
                <p className="text-right font-bold w-full">HieuVV</p>
            </div>
        </div>
    );
}