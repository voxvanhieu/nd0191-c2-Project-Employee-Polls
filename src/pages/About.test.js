import { render } from '@testing-library/react';
import { About } from './About';

describe('About page snapshot test', () => {
    it('will match snapshot', () => {
        let component = render(<About />);
        expect(component).toBeDefined();
        expect(component).toMatchSnapshot();
    })
})
