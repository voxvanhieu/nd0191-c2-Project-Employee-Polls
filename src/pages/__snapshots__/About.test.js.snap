// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page snapshot test will match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container mx-auto min-h-svh mt-8"
      >
        <div
          class="flex flex-col items-center justify-center text-center"
        >
          <h1
            class="text-5xl font-bold mt-20 mb-5 text-primary-600"
          >
            Udacity Lab Employee Polls
          </h1>
          <p
            class="text-lg text-justify"
          >
            The final lab exercise of Udacity's React Nanodegree is a comprehensive project that encapsulates the core principles of React and Redux in building a dynamic and interactive voting portal. In this application, users can log in with their personal accounts to engage in a community-driven polling environment. They have the ability to post new polls, reflecting current topics or queries they are curious about, and vote on existing ones, contributing to the collective decision-making process. The user interface is designed to be intuitive, allowing users to easily navigate between polls they have participated in and those they have yet to explore. The polls are sorted by creation date, ensuring a seamless and up-to-date experience. This project not only tests the students' ability to implement state management and component-based architecture but also their skills in creating a user-friendly platform that encourages active participation and interaction within the company's ecosystem.
          </p>
          <p
            class="text-right font-bold w-full"
          >
            HieuVV
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container mx-auto min-h-svh mt-8"
    >
      <div
        class="flex flex-col items-center justify-center text-center"
      >
        <h1
          class="text-5xl font-bold mt-20 mb-5 text-primary-600"
        >
          Udacity Lab Employee Polls
        </h1>
        <p
          class="text-lg text-justify"
        >
          The final lab exercise of Udacity's React Nanodegree is a comprehensive project that encapsulates the core principles of React and Redux in building a dynamic and interactive voting portal. In this application, users can log in with their personal accounts to engage in a community-driven polling environment. They have the ability to post new polls, reflecting current topics or queries they are curious about, and vote on existing ones, contributing to the collective decision-making process. The user interface is designed to be intuitive, allowing users to easily navigate between polls they have participated in and those they have yet to explore. The polls are sorted by creation date, ensuring a seamless and up-to-date experience. This project not only tests the students' ability to implement state management and component-based architecture but also their skills in creating a user-friendly platform that encourages active participation and interaction within the company's ecosystem.
        </p>
        <p
          class="text-right font-bold w-full"
        >
          HieuVV
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
